packages/labs/src/multi-select/multiSelect.tsx

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
i think both these files belong in the `select/` directory as they share interfaces and concepts